src/System.Management.Automation/engine/CommandDiscovery.cs

```diff
@@ -1113,7 +1113,7 @@ private static CommandInfo InvokeCommandNotFoundHandler(string commandName, Exec
                             }
                         }
 
-                        // TODO: this causes AppVeyor builds to fail due to invalid XML being output
+                        // TODO: this causes CI builds to fail due to invalid XML being output
```

Comment:
I think this comment should NOT be changed, since it was an AppVeyor specific issue. One should check if the problem still persist on Azure DevOps build infrastructure. Maybe even open a new issue to track this work.